Cargo.toml

```diff
@@ -14,6 +14,7 @@ rand = "0.4"
 serde = "1.0"
 serde_json = "1.0"
 serde_derive = "1.0"
+sha2 = "0.7.1"
```

Comment:
let's pin this to `"0.7"` so that we can pull in bug fixes without needing to push out a new release of this crate each time